xml.js

```diff
@@ -336,6 +336,9 @@ export function parse(source) {
   let children = null;
   let root = null;
   let stack = [];
+  if (source){
+    cleanupSVG(source)
```

Comment:
shouldn't it be this?
```suggestion
    source = cleanupSVG(source)
```

Reply:
I was between that one and just call svg-safe `validate`

Reply:
["parse, don't validate"](https://lexi-lambda.github.io/blog/2019/11/05/parse-don-t-validate/)